google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQuery.java

```diff
@@ -77,6 +77,40 @@ public String getSelector() {
     }
   }
 
+  enum DatasetView {
+    FULL("full"),
```

Comment:
Can we define these using the casing the service describes (e.g. uppercase), rather than performing case manipulation in the toString()?